themes/theme-b2c-tailwind/src/molecules/alert.css

```diff
@@ -0,0 +1,3 @@
+[data-store-alert] {}
+
+[data-store-alert-button] {}
```

Comment:
Why is it necessary? Considering there aren't any styles being specified

Reply:
I was wondering if I should implement a basic layout here. I was trying to get used to the styling workflow. 
I will implement a basic style or just remove these lines :)